styles/styles.css

```diff
@@ -95,48 +136,68 @@ body {
   line-height: 24px;
   color: #7f8cff;
   max-width: 250px;
+  padding-left: 24px;
 }
 
 .iconlists {
   display: flex;
   align-items: center;
   margin: 0;
-  padding: 15px 0;
+  margin-right: 300px;
+  padding: 15px 0 0 24px;
 }
 
 .iconlists li {
   list-style: none;
   margin-right: 20px;
 }
 
+/* Desktop - header */
+@media screen and (min-width: 768px) {
+  .header-main {
```

Comment:
- [ ] You are not supposed to duplicate the same line of code multiple times, this will not make your code clean and DRY, since you have this media query in line `51`, I would suggest you write this code in one media query and remove the other ones

Reply:
Hi @joskalenda, I have moved all the media query codes to a single block. Thanks for the feedback.